db/seeds.rb

```diff
@@ -37,5 +46,47 @@
   email: "jomo@example.com",
   password: "123456789", # high seructity!
   password_confirmation: "123456789",
-  role: Role.get(:superadmin)
-)
\ No newline at end of file
+  role: Role.get(:superadmin),
+  badge: Badge.get(:donor),
+  confirmed: true
+)
+User.create!(
```

Comment:
What's up with all of these? Honestly I'd much prefer removing all of the User seeds except the Steve one and require manual creation of the admin (or write a rake task that asks for user/pass)